ui.frontend/src/main/components/App/App.js

```diff
@@ -38,8 +38,8 @@ const App = () => {
     const { storeView, graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
     const { mountingPoints, pagePaths } = partialConfig;
     const config = {
-        ...partialConfig,
         storeView,
+        ...partialConfig,
```

Comment:
Is there a reason to change the order of `storeView` and `...partialConfig` here? :)

Reply:
Nope - I put it back just now.